GoogleForCreators/web-stories-wp
# Editor: Enable video optimization in pre-publish checklist by default

## Context

Now that video optimization has been worked out a bit - we know that we don't need the video optimization panel in the design panel. That is being removed and the experiment is being turned on.

## Summary

- Remove video optimization panel
- Enable experiment

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

Video Optimization panel is now gone. It will not show up when a video is selected.

## Testing Instructions


0. Make sure the `Video Optimization` setting is **not** checked in the dashboard settings
1. Editor - Add a video that needs to be optimized (large video - like 4k).
2. Click on video - open design panel. Should not see a `Video Processing` panel
3. Open the pre publish checklist - should see the toggle in the recommended section that asks to optimize videos

### QA






### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #7393
